Common/Orders/Order.cs

```diff
@@ -50,20 +50,20 @@ public abstract class Order
         /// <summary>
         /// Brokerage Id for this order for when the brokerage splits orders into multiple pieces
         /// </summary>
-        public List<string> BrokerId { get; internal set; }
+        public List<string> BrokerId { get; set; }
```

Comment:
Believe we should be able to avoid all the access changes using the correct constructor 👍

Reply:
I tried, but some of `ctor`s doesn't support some of required props.
For example there is no param for `Status`, but we need set the value during `Brokerage.GetOpenOrders`